### PR TITLE
Tv mode 2

### PR DESCRIPTION
**Pull request by dudethatbe**
This isn't quite working yet but the idea is to have a quick way to turn viz on and off.
currently, it just hides it, so there's still data coming in and out if there's a video being streamed.
I also added another option to change the size of the video. It's a WIP, but I want to position the video to fit between the top and bottom bars. Right now it makes the video smaller and puts it at the top right

### CODE REVIEW

**Comment by dudethatbe** [comment]
I've managed to get the video to fit between the bars, but it doesn't scale when the window resizes 🤔 the red border is for debugging

**Comment by dudethatbe** [comment]
I can't get the lint command to run locally yet, but I saw the error message here: 
```
Don't use computed properties with native classes. Use getters or @tracked properties instead  ember/no-computed-properties-in-native-classes
```
Makes sense to me even though I still don't fully get ember. I'll see if I can work around this.

File: app/components/datafruits-visuals.js [hunk 1]
```diff
@@ -1,16 +1,67 @@
...
+    }
+  }
+
   @service
   fastboot;
 
   @service
   videoStream;
 
+  @oneWay('videoStream.mode')
```
Comment (mcfiredrill) [inline app/components/datafruits-visuals.js:59]:
```suggestion
  get videoStreamMode() {
    return this.videoStream.mode;
  }
```
This should fix the lint error

File: app/components/datafruits-visuals.js [hunk 2]
```diff
@@ -1,16 +1,67 @@
...
+  videoStreamMode; // bg, tv
+
+  @oneWay('videoStream.displaying')
```
Comment (mcfiredrill) [inline app/components/datafruits-visuals.js:62]:
```suggestion
  get videoDisplaying() {
    return this.videoDisplaying;
  }
```

**Comment by dudethatbe** [comment]
ok i'm going to put the resizing on-hold and save that for another branch. This should simply give the option to turn the visuals on and off. Getting the video to adjust should use ember render modifiers https://github.com/emberjs/ember-render-modifiers#example-resizing-text-area

**Pull request merged**